app/styles/app.scss

```diff
@@ -22,6 +22,7 @@
 @import "app/base";
 @import "app/z-index";
 @import "app/typography";
+@import "app/heatmap";
```

Comment:
rename to `insights-heatmap` and put after line 99

Reply:
renamed to insights-heatmap and added after line 99